src/usage/BillingStatsPanel.tsx

```diff
@@ -55,6 +55,8 @@ const BillingStatsPanel: FC = () => {
   const today = new Date().toISOString()
   const dateRange = `${billingDateTime} UTC to ${today} UTC`
 
+  const [billingDate] = billingDateTime.split('T')
```

Comment:
since billingDateTime is returning the entire dateTime, we're just trimming it to the date